### ISSUE TITLE
Add docker image for easier deployment. 

### ISSUE THREAD

**Issue by jottr** [nprapps/portland2#90]
Title says it all. :)

**Comment by nakedsushi** [comment]
Would this work for you? https://hub.docker.com/r/scprdev/lunchbox/

**Comment by jottr** [comment]
Not really sure what this even is, or if it is, what it says in the readme...
There is no source Dockerfile linked on the Docker hub. 
I suggest that the Dockerfile should be part of this projects repo.

**Comment by nakedsushi** [comment]
Yep it's mine. I'm still working on it but right now it builds and runs. I have a pull request from https://github.com/SCPR/lunchbox if you want to test it out.

**Comment by jottr** [comment]
Ah, I see. Neat. I'll have a look.
